framework/src/main/java/org/checkerframework/common/value/ValueAnnotatedTypeFactory.java

```diff
@@ -1302,6 +1302,30 @@ private boolean methodIsStaticallyExecutable(Element method) {
 
         @Override
         public Void visitMethodInvocation(MethodInvocationTree tree, AnnotatedTypeMirror type) {
+            if ((getMethodIdentifier().isMathMin(tree, processingEnv)
+                            || getMethodIdentifier().isMathMax(tree, processingEnv))
+                    && type.hasAnnotation(UNKNOWNVAL)) {
+                AnnotatedTypeMirror atm1 = getAnnotatedType(tree.getArguments().get(0));
```

Comment:
I suggest giving these variables somewhat more descriptive names. I would choose `argType1` and `argType2`, but you might prefer something else.